src/modules/integrator-guide/pages/corteza-studio/index.adoc

```diff
@@ -0,0 +1,139 @@
+include::ROOT:partial$variables.adoc[]
+
+= {PRODUCT_NAME} Studio
+
```

Comment:
This is an exception; when you add page properties (the `:something: ...` bits), there is no empty line before that; so 

```
= Something
:description: ...
```

For other cases, there is an empty line